shared/constants/types/flux.js

```diff
@@ -19,4 +19,4 @@ export type AsyncAction = (dispatch: Dispatch, getState: GetState) => ?Promise<*
 export type Dispatch = (action: AsyncAction | Action) => ?Promise<*>
 
 export type TypedAsyncAction<A> = (dispatch: TypedDispatch<A>, getState: GetState) => ?Promise<*>
-export type TypedDispatch<A> = (action: TypedAsyncAction<A> | A) => ?Promise<*>
+export type TypedDispatch<-A> = (action: TypedAsyncAction<A> | A) => ?Promise<*>
```

Comment:
Necessary so that we can have `TypedAsyncAction`s dispatch other `TypedAsyncAction`s which dispatch a subtype of the original action's actions. :upside_down_face: